src/sentry/static/sentry/app/redux/actions/workDefinition.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+import {makeResourceActions} from './shared';
+
+export const RESOURCE_NAME = 'WORK_DEFINITION';
+
+export const workDefinitionActions = makeResourceActions(
+  RESOURCE_NAME,
+  '/api/0/organizations/{org}/NOT-IMPLEMENTED/', // available-work can return work definitions
```

Comment:
As general rule, I think it's best not to add things before they are used anywhere. Going back here in 3 months, it's very hard to know if the code should be saved or is outdated.

Reply:
Agreed. The actions is used. It's just the list endpoint that's not used. It's a hack though, and I was confused by this myself. Should just be null.